CovertInfiltration/Src/CovertInfiltration/Classes/UICovertActionsGeoscape.uc

```diff
@@ -1148,6 +1148,11 @@ simulated protected function TriggerTutorialOnSelection ()
 	{
 		class'UIUtilities_InfiltrationTutorial'.static.ActivityChains();
 	}
+
+	if (GetAction().bStarted)
+	{
+		class'UIUtilities_InfiltrationTutorial'.static.CovertOpsAbort();
+	}
```

Comment:
This will be triggered right after confirming the first ever CA. Feels a bit backwards regarding "Delay tutorials until relevant"